gocron.go

```diff
@@ -391,22 +394,16 @@ type Scheduler struct {
 }
 
 func (s *Scheduler) Len() int {
-	s.mu.Lock()
 	l := len(s.jobs)
-	s.mu.Unlock()
 	return l
 }
 
 func (s *Scheduler) Swap(i, j int) {
-	s.mu.Lock()
```

Comment:
My guess is that trying to do a lock in the middle of a sort is totally insance.

Reply:
So I've put a lock around anything that does a sort.